.github/workflows/checkE2ETestCode.yml

```
@@ -18,6 +18,8 @@ jobs:
 
       - name: Setup Node
         uses: ./.github/actions/composite/setupNode
+        with:
+          IS_DESKTOP_BUILD: 'true'
 
       - name: Verify e2e tests compile correctly
-        run: npm run e2e-test-runner-build
\ No newline at end of file
+        run: npm run e2e-test-runner-build
```

New Line should be at the end of all files
```suggestion
        run: npm run e2e-test-runner-build

```